menu/menu.yaml

```diff
@@ -140,7 +140,11 @@
         - Title: "Azure Service Bus Transport"
           Url: "nservicebus/azure-service-bus"
           Articles:
-            - "nservicebus/azure-service-bus/configuration"
+            - Title: "Azure Service Bus Transport"
+              Url: "nservicebus/azure-service-bus/configuration"
+              Articles:
+                - "nservicebus/azure-service-bus/configuration/full"
```

Comment:
Hmm, that's too bad... how do you suggest we work around this limit? both articles logicall sit beneath the configuration article, just no entry in the menu and reference from the file? Or flatten it?